### PR TITLE
Bug 2173525: Disk and net icons for boot order modal

### PR DESCRIPTION
**Pull request by upalatucci**
## 📝 Description

Adding icons for boot order modal

## 🎥 Demo

**Before**
Screenshot from 2023-04-11 10-06-29

**After**

image

### CODE REVIEW

File: src/utils/components/BootOrderModal/BootOrderModalBody.tsx [hunk 1]
```diff
@@ -68,6 +72,8 @@ export const BootOrderModalBody: React.FC<{
 
   const showEmpty = devices.length === 0 && !isEditMode;
 
+  console.log(devices);
```
Comment (pcbailey) [inline src/utils/components/BootOrderModal/BootOrderModalBody.tsx:75]: I think you forgot about this. =)

File: images/hard-drive-regular.svg
Comment (metalice) [inline images/hard-drive-regular.svg:1]: Was this icon received from UX?

Comment (upalatucci) [reply images/hard-drive-regular.svg:1]: Yes but we can change it if you want

File: src/utils/components/BootOrderModal/BootOrderModalBody.tsx [hunk 2]
```diff
@@ -105,9 +111,17 @@ export const BootOrderModalBody: React.FC<{
                             <DataListCell key={value.name}>
                               <Split>
                                 <SplitItem isFilled>
-                                  <span id={value.name}>
-                                    {value.name}
-                                    {` (${typeLabel})`}
+                                  <span id={value.name}>{value.name}</span>
+
+                                  <span className="pf-u-ml-sm">
+                                    {type === DeviceType.DISK ? (
```
[the remark this answers is not in the document]
Comment (upalatucci) [reply src/utils/components/BootOrderModal/BootOrderModalBody.tsx:117]: Do we have to support more boot sources types in the future?

File: src/utils/components/BootOrderModal/DeviceTypeIcon.tsx [hunk 3]
```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+
+import { DeviceType } from '@kubevirt-utils/resources/vm/utils/boot-order/bootOrder';
+import { NetworkIcon, VolumeIcon } from '@patternfly/react-icons';
+
+type DeviceTypeIconProps = {
+  type: DeviceType;
+};
+
+export const DeviceTypeIconMapping = {
```
Comment (metalice) [inline src/utils/components/BootOrderModal/DeviceTypeIcon.tsx:10]: why export? maybe add a default value? why as const needed?

Comment (upalatucci) [reply src/utils/components/BootOrderModal/DeviceTypeIcon.tsx:10]: I can remove the export and add a default value. The Volume Icon probably.
`as const` is useful in typescript as it allows to tried it just like an enum

File: src/utils/components/BootOrderModal/boot-order.scss [hunk 4]
```diff
@@ -1,35 +1,32 @@
...
-  padding-top: var(--pf-global--spacer--md);
-  padding-left: var(--pf-global--spacer--lg);
-  padding-right: var(--pf-global--spacer--lg);
-}
+  &__hard-disk-icon {
+    width: 18px;
+    height: 14px;
+    position: relative;
+    top: 2px;
+  }
```
Comment (avivtur) [inline src/utils/components/BootOrderModal/boot-order.scss:23]: is this used somewhere? if not please remove

**Review by avivtur — COMMENTED** [review]
looks good, just don't forget to remove the svg file if not used

File: src/utils/components/BootOrderModal/boot-order.scss (hunk 4 above)
Comment (upalatucci) [reply src/utils/components/BootOrderModal/boot-order.scss]: right

**Review by avivtur — APPROVED** [review]

**Pull request merged**